src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDurationMappingTest.java

```diff
@@ -0,0 +1,371 @@
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.BDDMockito.given;
+
+public class HearingsDurationMappingTest  extends HearingsMappingBase {
+    @Mock
+    private HearingDurationsService hearingDurations;
+
+    @Mock
+    private ReferenceDataServiceHolder referenceDataServiceHolder;
+
+    @DisplayName("when a valid adjournCaseDuration and adjournCaseDurationUnits is given getHearingDuration returns the correct duration Parameterized Tests")
+    @ParameterizedTest
+    @CsvSource(value = {
+        "120,minutes,120",
+        "1,sessions,165",
+    }, nullValues = {"null"})
+    void getHearingDuration(String adjournCaseDuration, String adjournCaseDurationUnits, int expected) {
+
+        SscsCaseData caseData = SscsCaseData.builder()
+            .benefitCode(BENEFIT_CODE)
+            .issueCode(ISSUE_CODE)
+            .adjournCaseNextHearingListingDurationType("nonStandardTimeSlot")
+            .adjournCaseNextHearingListingDuration(adjournCaseDuration)
+            .adjournCaseNextHearingListingDurationUnits(adjournCaseDurationUnits)
+            .appeal(Appeal.builder()
+                        .hearingOptions(HearingOptions.builder().build())
+                        .build())
+            .build();
+        int result = HearingsDurationMapping.getHearingDuration(caseData, referenceDataServiceHolder);
+
+        assertEquals(expected, result);
+    }
+
+
+    @DisplayName("when a invalid adjournCaseDuration or adjournCaseDurationUnits is given getHearingDuration returns the default duration Parameterized Tests")
+    @ParameterizedTest
+    @CsvSource(value = {
+        "null,null",
+        "null,60",
+        "1,test",
+    }, nullValues = {"null"})
+    void getHearingDuration(String adjournCaseDuration, String adjournCaseDurationUnits) {
+        // TODO Finish Test when method done
+        given(hearingDurations.getHearingDuration(BENEFIT_CODE, ISSUE_CODE))
+            .willReturn(new HearingDuration(BenefitCode.PIP_NEW_CLAIM, Issue.DD,
+                                            60, 75, 30
+            ));
+
+        given(referenceDataServiceHolder.getHearingDurations()).willReturn(hearingDurations);
+        SscsCaseData caseData = SscsCaseData.builder()
+            .benefitCode(BENEFIT_CODE)
+            .issueCode(ISSUE_CODE)
+            .adjournCaseNextHearingListingDurationType("nonStandardTimeSlot")
+            .adjournCaseNextHearingListingDuration(adjournCaseDuration)
+            .adjournCaseNextHearingListingDurationUnits(adjournCaseDurationUnits)
+            .appeal(Appeal.builder()
+                        .hearingOptions(HearingOptions.builder().build())
+                        .build())
+            .build();
+        int result = HearingsDurationMapping.getHearingDuration(caseData, referenceDataServiceHolder);
+
+        assertEquals(30, result);
```

Comment:
Personally I prefer `assertThat(x).isEqualTo(y)`

You have used that below in this file, so it would be better to pick one and be consistent.

Reply:
That is the thing when you move stuff from one file to the next you become the author. Everyone else's design decisions become yours. I will think on this.

Reply:
I have gone with the majority of examples and chosen assertThat